pom.xml

```
@@ -340,6 +340,28 @@
                             <modelNameSuffix>dto</modelNameSuffix>
                         </configuration>
                     </execution>
+                    <execution>
+                        <id>payment-methods-api-v2</id>
+                        <goals>
+                            <goal>generate</goal>
+                        </goals>
+                        <configuration>
+                            <inputSpec>https://raw.githubusercontent.com/pagopa/pagopa-infra/CHK-2798-openapi-multi-fee/src/domains/ecommerce-app/api/ecommerce-payment-methods-service/v2/_openapi.json.tpl</inputSpec>
```

```suggestion
                            <inputSpec>https://raw.githubusercontent.com/pagopa/pagopa-infra/main/src/domains/ecommerce-app/api/ecommerce-payment-methods-service/v2/_openapi.json.tpl</inputSpec>
```
TODO to be replaced after dependent infra PR merge